xprof/btx_tally.c

```diff
@@ -0,0 +1,17 @@
+#include <babeltrace2/babeltrace.h>
+#include "tally.h"
```

Comment:
Maybe rename it to `btx_tally` then?

Reply:
Makes sense, since we have babeltrace code spread on these files. I understand proper babeltrace stuff should be btx_.....